numba_dppy/compiler.py

```diff
@@ -31,12 +31,11 @@
 
 from . import spirv_generator
 
-import os
```

Comment:
We have a lot of places with incorrect imports. Need to apply isort and pylint in CI.

Reply:
https://github.com/IntelPython/numba-dppy/issues/335